item-spec/common-metadata.md

```diff
@@ -0,0 +1,136 @@
+# STAC Common Metadata
+
+* [Date and Time](#date-and-time)
+* [Licensing](#licensing)
+* [Provider](#provider)
+* [Instrument](#instrument)
+* [Metadata](#metadata)
+
+Various *examples* are available in the folder [`examples`](examples/).
+*JSON Schemas* can be found in the folder [`json-schema`](json-schema/).
+
+## Date and Time
+
+Fields to provide additional temporal information such as ranges with a start and an end datetime stamp.
+
+### Date and Time Range
+
+- [JSON Schema](json-schema/datetimerange.json)
+
+While a STAC item can have a nominal datetime describing the capture, this extension allows an item to have a range
+of capture datetimes. An example of this is the [MODIS 16 day vegetation index product.](https://lpdaac.usgs.gov/products/mod13q1v006/).
+The datetime property in a STAC item and these fields are not mutually exclusive.
+
+**Important:** Using one of the fields REQUIRES to include the other fields as well to enables a user to search STAC records by the provided times.
```

Comment:
what does "the other fields" refer to here?

Reply:
If you use start_datetime you need to add end_datetime, too. And vice-versa.